SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -153,7 +153,18 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 			Connection->ReceptionistConfig.ReceptionistPort = LoadedWorld->URL.Port;
 		}
 
-		Connection->ReceptionistConfig.UseExternalIp = LoadedWorld->URL.HasOption(TEXT("useExternalIpForBridge"));
+		FString UseExternalIpOption = FString(LoadedWorld->URL.GetOption(TEXT("useExternalIpForBridge"), TEXT("")));
```

Comment:
This is a boolean option no? So this should already be correct. 
The existence of the option itself indicates the boolean value.
`"/Maps/MyMap?useExternalIpForBridge"` would set `ReceptionistConfig.UseExternalIp = true`
`"/Maps/MyMap"` would set `ReceptionistConfig.UseExternalIp = false`

Don't think this change is needed.